### ISSUE TITLE
Whitelist

### ISSUE THREAD

**Issue by morrisj95** [morrisj95/spam-driven-recruiting#1]
It has come to my attention that often, reasonably good companies to work at will have their hiring process confined to going through recruiting agencies -- sometimes the same ones we might list here.

Anyone with knowledge of these practices should create a whitelist document and add their entries that listt he company name and the recruiting agencies they use.

Reason: Person wants to work for Company X, but they force hiring through a recruiting company they added to their block list from the main document. Person now cannot work for company X or will be oblivious to attempts at communication.

**Comment by dabockster** [comment]
I understand where you're going with this. But think about the reasons why this repo was created in the first place. Going by your example, if "Company X" is dead set on using a recruiting firm that we've blacklisted as a community, then the statistical probability of the same company being, frankly, a total shitshow, is high enough that it would set off the spider senses of any reasonable person.

Maybe having documents where reasons are given for every blacklisted domain would be more appropriate. Then people could make better educated decisions for themselves.

**Comment by morrisj95** [comment]
Makes sense.  I sort of touch on that with the pull request "template" and my latest commit message's extended notes.  

I think a wiki would be a good, organized way to do it, but I want to avoid the way a wiki goes unchecked.  Without proper checks it's too easy to become a flame board or have copy/pasted emails without personal details removed -- which could open up legal issues.

**Comment by morrisj95** [comment]
Similarly, the requirement for reasons/examples when contributing are there to avoid blacklisting perfectly fine recruiting agencies

**Comment by dabockster** [comment]
I've been thinking about starting a community website about CS hiring for a while now. Something completely independent from major social media, GitHub, and Reddit. Good moderation and filtering as well. But I'm dealing with some mental issues right now, so I haven't been able to assemble the tools I need to make that happen.
